samples/test/util.py

```diff
@@ -17,9 +17,9 @@
 # Add **kwargs, so that when new arguments are added, this doesn't fail for
 # unknown arguments.
 def _default_verify_func(
-    run_id: int, run: kfp_server_api.ApiRun,
-    mlmd_connection_config: metadata_store_pb2.MetadataStoreClientConfig,
-    **kwargs
+        run_id: int, run: kfp_server_api.ApiRun,
```

Comment:
It seems that your IDE has a different formatting than we use. In repo root, there is a config to use yapf to format python code. Can you try to configure yapf as your IDE formatter

Reply:
Goland can't customize yapf config. It use default yapf format. I will switch to vs code later.